code/modules/jobs/job_types/assistant.dm

```diff
@@ -18,6 +18,8 @@ Assistant
 	paycheck_department = ACCOUNT_CIV
 	display_order = JOB_DISPLAY_ORDER_ASSISTANT
 
+	job_family_heirlooms = list(/obj/item/storage/toolbox/mechanical/old/heirloom, /obj/item/clothing/gloves/cut/heirloom)
```

Comment:
Just change this to `family_heirlooms`, we know it's the job's heirlooms because it's in the job datum.